openslides/core/static/js/core/base.js

```diff
@@ -808,6 +808,7 @@ angular.module('OpenSlidesApp.core', [
                                 id: value.id,
                             };
                         } else if (
+                            value.name != 'voting/prompt' &&
```

Comment:
Whats about changing this from a blacklist to a white list allowing just the named forms (like `motion/motions`) to return an `EditForm`?
But we need a hook for plugins here:
1) We have a blacklist: Plugins need to set them on there, because if this is not possible, you will get the error you described. (!)
2) We have a whitelist: A Plugin may have EditForms, so they want to hook those in here.
@emanuelschuetze ?